innersource-ambassador.md - open source experience + motivation of ambassadors

sallydeering1 opened the pull request:
Under Solutions, added note about also ideally having open source experience.  Under Rationale, added information on what's in it for the ambassadors.

sallydeering1 committed:
Update innersource-ambassador.md

Under Solutions, added note about also ideally having open source experience.  Under Rationale, added information on what's in it for the ambassadors.

spier commented on patterns/1-initial/innersource-ambassador.md:
```diff
@@ -50,7 +50,7 @@ In a multinational organization, the ISPO implemented InnerSource initiatives bu
 ## Rationale
 
-InnerSource Ambassadors leverage their existing knowledge of their departments and relationships within the organization to address challenges the ISPO cannot solve alone. This decentralization of responsibility enhances the ISPO's reach while maintaining centralized oversight.
+InnerSource Ambassadors leverage their existing knowledge of their departments and relationships within the organization to address challenges the ISPO cannot solve alone. This decentralization of responsibility enhances the ISPO's reach while maintaining centralized oversight.  The rewards for the Ambassadors will vary by individual motivations.  Rewards that ambassadors have expressed are that  ...
```
```suggestion
InnerSource Ambassadors leverage their existing knowledge of their departments and relationships within the organization to address challenges the ISPO cannot solve alone. This decentralization of responsibility enhances the ISPO's reach while maintaining centralized oversight.

The rewards for the ambassadors will vary by individual motivations. Rewards that ambassadors have expressed are that being an ambassador provides opportunity to build their internal network, enhance their communication and influence skills, build their knowledge further about InnerSource, build their presentation and speaking skills, and learn about other products and teams within their company.
```

splitting this up into two paragraphs

Also changed InnerSource to our [preferred](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/meta/innersource-spelling.md) spelling

spier committed:
Update patterns/1-initial/innersource-ambassador.md

splitting this up into two paragraphs

Also changed InnerSource to our preferred spelling
